storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/put/DefaultPutResolver.java

```diff
@@ -75,14 +82,14 @@ private PutResult insert(@NonNull StorIOSQLiteDb storIOSQLiteDb, @NonNull Conten
     private PutResult updateOrInsert(@NonNull StorIOSQLiteDb storIOSQLiteDb,
                                      @NonNull ContentValues contentValues,
                                      @NonNull String table,
-                                     @NonNull final String idFieldName,
-                                     @NonNull Object id) {
+                                     @NonNull String idFieldName,
+                                     @NonNull String id) {
 
         final int numberOfUpdatedRows = storIOSQLiteDb.internal().update(
                 new UpdateQuery.Builder()
                         .table(table)
                         .where(idFieldName + "=?")
-                        .whereArgs(String.valueOf(id))
```

Comment:
`String.valueOf(object)` will return `String` `"null"` if object will be null, which is not very obvious behaviour for user :)